lib/core/service/shared_preference_service/shared_preference_service.dart

```diff
@@ -0,0 +1,9 @@
+import 'package:firebase_auth/firebase_auth.dart';
+
+abstract class SharedPreferenceService {
+  Future saveLoginDetails({required UserCredential user});
```

Comment:
missing expected return type `Future` is generic you should always include generic type.
ex: `Future<bool>`, `Future<String>` or Future<void> if no return type

Reply:
oks sir. na change n nhu